MobiFlight/Joysticks/Joystick.cs

```diff
@@ -167,19 +131,19 @@ protected virtual void EnumerateDevices()
                     // when looking up names in the the .joystick.json file.
                     var buttonName = $"{ButtonPrefix} {device.Usage}";
                     var buttonLabel = MapDeviceNameToLabel(buttonName);
-                    Buttons.Add(new JoystickDevice() { Name = buttonName, Label = buttonLabel, Type = JoystickDeviceType.Button });
+                    Buttons.Add(new JoystickDevice() { Name = buttonName, Label = buttonLabel, Type = DeviceType.Button });
```

Comment:
That's fine. a button is a button, also on a Joystick.

Reply:
Added extra JoystickDeviceType.